images/native-post.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+#
+# This is a post install script that is run from the config after the image
+# has been created by clr-installer for images with special requirements
+# ex: <image_name>-post.sh
+
+set -e
+
+file_name=$(basename "$0")
+
+echo "Hello World from ${file_name}"
```

Comment:
Does this get output to the clr-installer logs?  Looking to see if there's a good way to tell if this ran.

Reply:
it does not print anything on console.
clr-installer must have control over it.

Reply:
maybe "wait to boot" would be a better example